marklogic-data-hub-central/ui/src/assets/mock-data/user-context-mock.ts

```diff
@@ -9,7 +9,7 @@ const defaultUserData: UserContextInterface = {
     type: ''
   },
   pageRoute: '/',
-  maxSessionTime: 300
+  maxSessionTime: 600
```

Comment:
Seems like this would be worth making into a constant somewhere so it doesn't have to be changed in many places the next time we change this value

Reply:
Good idea; updated the PR